src/client/package.json

```diff
@@ -9,16 +9,19 @@
         "@testing-library/jest-dom": "^4.2.4",
         "@testing-library/react": "^9.5.0",
         "@testing-library/user-event": "^7.2.1",
+        "jsonwebtoken": "^8.5.1",
         "lodash": "^4.17.20",
+        "luxon": "^1.26.0",
```

Comment:
we are using moment.js to deal with date time

Reply:
I was mostly using for debugging - will review.  
Given [Moment Project Status](https://momentjs.com/docs/#/-project-status/) we should probably review at some point.

Reply:
Was using for (commented-out) console logging. Commented out import and removed reference in package.json